CHANGELOG.md

```diff
@@ -1,21 +1,37 @@
 # Digital Marketplace front-end toolkit changelog
 
 Records breaking changes from major version bumps
-## 23.0.0
 
-PR
+## 24.0.0
 
-## What changed
+### What changed
+
+- 'media-down' IE helper mixin has been moved to a '_conditionals2.scss' stylesheet, which is consistent
+  with its location in the equivalent gov.uk repository;
+- this means we can use it even if we aren't using the next/previous navigation stylesheet.
+- non-breaking change: "report-a-problem" JS for feedback forms feature.
+
+###
+
+Example app change:
+
+app/assets/scss/application.scss
+```diff
++@import "toolkit/shared_scss/_conditionals2.scss";
+```
+
+## 23.0.0
+
```

Comment:
Looks like the PR was never included here. It's https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/353. Would be nice to pop it in.